### PR TITLE
Update moment month translations

### LINKED ISSUE #2173: Update moment month translations
**Is your feature request related to a problem? Please describe.**
The translations for the date, as used by the BBC, were stored partly in `PAL` and partly in Moment under the BBC specific `moment-ws`. 

As we now only want to use `moment`, we have to apply the correct date translations.

**Describe the solution you'd like**
Apply the correct month values as described in the spreadsheet. Where possible the month string has been created from the PAL config. As we only use the full month description on the Frontpage, only these values will be updated.

- Afaan Oromoo -> https://github.com/bbc/psammead/pull/2187
- Amharic -> https://github.com/bbc/psammead/pull/2187
- Bengali -> https://github.com/bbc/psammead/pull/2227
- Gahuza -> https://github.com/bbc/psammead/pull/2193
- Gujarati -> https://github.com/bbc/psammead/pull/2227
- Hausa -> https://github.com/bbc/psammead/pull/2196
- Hindi -> https://github.com/bbc/psammead/pull/2227
- ~~Igbo~~ `Doesn't require updating - according to spreadsheet on WSFRONTPGS-200`
- Nepali -> https://github.com/bbc/psammead/pull/2210
- Pashto -> https://github.com/bbc/psammead/pull/2227
- Somali -> https://github.com/bbc/psammead/pull/2196
- Swahili -> https://github.com/bbc/psammead/pull/2227
- Tigrinya -> https://github.com/bbc/psammead/pull/2193
- ~~Ukrainian~~
`We do not need to override months for Ukrainian for now because those are Russian and are required for a Russian locale.`
- Urdu

More details WSFRONTPGS-200

**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [X] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by Bopchy**
Resolves #2173 

**Overall change:** _Update moment month translations not updated in other PRs._

**Code changes:**

- _Create Bengali, Swahili and Urdu locales, with updated moment month translations._
- _Update Gujarati and Hindi moment month translations._
- _Confirm Pashto month translations are correct._
- _Add tests for created locales._
- _Update tests._

**Expected changes**
- On storybook, the various months for the services should be:

| Service | Month translations |
|---------|--------------------|
| Bengali | নুয়ারী , ফেব্রুয়ারি , মার্চ , এপ্রিল , মে , জুন , জুলাই , অগাস্ট , সেপ্টেম্বর , অক্টোবর , নভেম্বর , ডিসেম্বর |
| Swahili | Januari , Februari , Machi , Aprili , Mei , Juni , Julai , Agosti, Septemba, Oktoba, Novemba, Disemba |
| Urdu (rtl) | جنوری , فروری , مار چ , اپريل , مئ , جون , جولائی , اگست , ستمبر , اکتوبر , نومبر , دسمبر |
| Gujarati | જાન્યુઆરી, ફેબ્રુઆરી, માર્ચ, એપ્રિલ, મે, જૂન, જુલાઈ, ઓગસ્ટ, સપ્ટેમ્બર, ઑક્ટોબર, નવેમ્બર, ડિસેમ્બર |
| Hindi | जनवरी ,फ़रवरी ,मार्च ,अप्रैल ,मई ,जून ,जुलाई ,अगस्त ,सितंबर ,अक्टूबर ,नवंबर ,दिसंबर |

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/utilities/psammead-locales/moment/sw.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+var moment = require('moment');
+require('moment/locale/sw');
+
+moment.updateLocale('bn', {
```
Comment (ghost) [inline packages/utilities/psammead-locales/moment/sw.js]:
```suggestion
moment.updateLocale('sw', {
```

Comment (AlistairGempf) [reply packages/utilities/psammead-locales/moment/sw.js]: There is similar in the `urdu` file too.

**Review by ghost — APPROVED** [review]

File: packages/utilities/psammead-locales/moment/index.stories.jsx [hunk 2]
```diff
@@ -64,12 +68,14 @@ const locales = [
...
   { name: 'Serbian Cyrillic', locale: 'sr-cyrl' },
-  { name: 'Somali', locale: 'so' },
   { name: 'Sinhala', locale: 'si' },
+  { name: 'Somali', locale: 'so' },
+  { name: 'Swahili', locale: 'sw' },
   { name: 'Tamil', locale: 'ta' },
   { name: 'Thai', locale: 'th' },
   { name: 'Tigrinya', locale: 'ti' },
   { name: 'Ukrainian', locale: 'uk' },
+  { name: 'Urdu', locale: 'ur' },
```
Comment (ghost) [inline packages/utilities/psammead-locales/moment/index.stories.jsx]:
```suggestion
  { name: 'Urdu', locale: 'ur' , dir: 'rtl' },
```

**Review by EinsteinNjoroge — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by AlistairGempf — APPROVED** [review]
Looks good.

**Review by j-pendlebury — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**